src/clojuratica/lib/options.clj

```diff
@@ -1,4 +1,5 @@
-(ns clojuratica.lib.options)
+(ns clojuratica.lib.options

```

Comment:
again with explicit opts passing this ns could be simplified and much code deleted